app.rb

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env ruby
+
+require 'sinatra'
+require 'sinatra/json'
+require 'oauth2'
+require 'manifoldco_signature'
+
+# Manifold's public key (or your local test version), used for by the
+# manifoldco_signature gem to verifiy that requests came from Manifold.
+set :verifier_key, ENV['VERIFIER_KEY'] || ManifoldcoSignature::MASTER_KEY
```

Comment:
It'd be more inline with what this key is suppose to represent, you pass the masterkey into the verifier.